src/cmd4.cc

```diff
@@ -3801,7 +3773,6 @@ void do_cmd_knowledge(void)
 		prt("(7) Display current pets", 10, 5);
 		prt("(8) Display current quests", 11, 5);
 		prt("(9) Display current fates", 12, 5);
-		prt("(0) Display known traps", 13, 5);
```

Comment:
Lucky that this just so happens where the absence of a "0" option won't be missed :)